crates/bevy_app/src/app.rs

```diff
@@ -50,10 +51,13 @@ fn run_once(mut app: App) {
 }
 
 impl App {
+    /// Returns an [`AppBuilder`] with a basic structure that can be configured further
+    /// to create an `App`.
     pub fn build() -> AppBuilder {
         AppBuilder::default()
     }
 
+    /// Updates the [`Schedule`] of the `App`.
```

Comment:
"Runs" would probably be better here?

Reply:
"Runs" gives the idea of something that continues over time, while this is just a tick thing. Maybe "Executes" could be even better?

Reply:
Maybe "Advances the Schedule by one iteration" is more explicit...

Reply:
Hmm, maybe. I just don't think using the word "update" to describe a function named "update" is a very nice idea.

Reply:
Well, actually you're right. Maybe we can go with "[verb] the `Schedule` by/for one cycle", where [verb] is "Executes", "Advances" or "Runs". Then the cyclic nature of the schedule can be better described on the docs of the `Schedule`.